react/features/salesforce/useSalesforceLinkDialog.ts

```diff
@@ -40,7 +41,7 @@ export const useSalesforceLinkDialog = () => {
     const [ hasDetailsErrors, setDetailsErrors ] = useState(false);
     const conference = useSelector(getCurrentConference);
     const sessionId = conference?.getMeetingUniqueId();
-    const { salesforceUrl = '' } = useSelector((state: IReduxState) => state['features/base/config']);
+    const salesforceUrl = useSelector((state: IReduxState) => getSalesforceUrl(state) ?? '');
```

Comment:
```suggestion
    const salesforceUrl = useSelector(getSalesforceUrl);
```

Reply:
Why do we need a default value here? Make `getSalesforceUrl` always return a string then?